src/main/java/org/myrobotlab/service/DiyServo.java

```diff
@@ -138,11 +138,11 @@ public void run() {
 
                   // ok targetPos is reached ( with tolerance )
                   if (nbSamePosInputSinceX >= 3 || getTargetOutput() == targetPos) {
-                    publishServoData(SERVO_EVENT_STOPPED, getTargetOutput());
+                    publishServoEvent(SERVO_EVENT_STOPPED, getTargetOutput());
                   } else {
 
                     if (getTargetOutput() != targetPos) {
-                      publishServoData(SERVO_EVENT_POSITION_UPDATE, getTargetOutput());
+                      publishServoEvent(SERVO_EVENT_STARTED, getTargetOutput());

```

Comment:
This is wrong.. this isn't when it starts moving.  this is when it's updating it's position.  remove.